lib/minify.js

```diff
@@ -54,6 +54,7 @@ function minify(files, options) {
                 cache: null,
                 eval: false,
                 ie8: false,
+                safari10: false,
```

Comment:
Can you move this down above `toplevel` so as to keep this section in alphabetical order?